pages/index.vue

```diff
@@ -2,7 +2,7 @@
   <div class="flex grow justify-center p-5">
     <div class="w-full rounded border border-gray-300 p-5">
       <h2 class="text-center text-2xl font-bold">
-        {{ `Welcome` }}
+        {{ $t('welcome', { name: 'vue-i18n' }) }}
```

Comment:
What's the name field for here? Is this just for string interpolation so it could say "Welcome, <USERNAME>"?

Reply:
That was me testing something and forgetting to remove it, the name field isn't necessary.